cdap-docs/developers-manual/source/building-blocks/workflows.rst

```diff
@@ -185,6 +185,89 @@ The diagram for this code would be:
    :width: 8in
    :align: center
 
+Conditional Node
```

Comment:
Unrelated to this PR, but on the very first line of this page, s/thousand/thousands/